client/src/utils/timeout.ts

```diff
@@ -0,0 +1,7 @@
+let timeoutHandler: ReturnType<typeof setTimeout> | null = null;
+export const timeout = (callback: () => void, delay = 5000) => {
+    if (timeoutHandler) {
+        clearTimeout(timeoutHandler);
+    }
+    timeoutHandler = setTimeout(callback, delay);
+};
```

Comment:
This utility is prone to breaking if it is used in more than one place anywhere, even by multiple instances of GridList. It looks like this could maybe be handled by [useDebounceFn](https://vueuse.org/shared/useDebounceFn) ?

If it is supposed to be overarching across GridList instances, please move it to the GridList folder, or make it a store.

Reply:
Good idea, thanks. I opened a PR to use debounce instead of the custom timeout helper. #17057